examples/computer_vision/byol_pytorch/data.py

```diff
@@ -0,0 +1,301 @@
+from attrdict import AttrDict
+from dataclasses import dataclass
+from enum import auto, Enum
+import os
+import random
+from typing import Callable, List, Tuple
+
+import torch
+import torch.nn as nn
+from torchvision.datasets import CIFAR10, ImageFolder, STL10
+import torchvision.transforms as T
+from torch.utils.data import Dataset
+
```

Comment:
i like what you did here.  let's keep this in mind for the future if we work on utility libraries.